Classes/Command/CrawlerCommandController.php

```diff
@@ -69,6 +70,36 @@ class CrawlerCommandController extends CommandController
      */
     protected $siteRepository;
 
+    /**
+     * @Flow\Inject
+     * @var DomainRepository
+     */
+    protected $domainRepository;
+
+
+    /**
+     * Crawl all sites
+     *
+     * @throws EelException
+     * @throws MissingActionNameException
+     * @throws PropertyException
+     * @throws NeosException
+     * @throws HttpException
+     */
+    public function crawlSitesCommand(): void {
+        $sites = $this->siteRepository->findAll();
+        $this->outputLine('Found %d sites', [count($sites)]);
+
+        foreach ($sites as $site) {
+            $siteNodeName = $site->getNodeName();
+            $domain = $this->domainRepository->findOneBySite($site, true);
+            $urlSchemeAndHost = ($domain->getScheme() || 'http') . '://' . $domain->getHostname() . ($domain->getPort() ? ':' . $domain->getPort() : '');
```

Comment:
Would be good to check if a domain has been found here and skip the crawl with some output if not.

Reply:
Oh yes, this makes sense!